examples/pip_install_incremental/WORKSPACE

```diff
@@ -0,0 +1,40 @@
+workspace(name = "example_repo")
+
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+# TODO(hrfuller) change this to a released http_archive when a release exists.
```

Comment:
nit: in rules_nodejs we usually just let the example go ahead of the repo a bit, with the assumption that releases are frequent enough. You don't want all these examples features to require a second PR to patch up what the first one did

The integration test runner always swaps in the current code anyway. It expects the releases/download URL
https://github.com/bazelbuild/rules_python/blob/master/tools/bazel_integration_test/test_runner.py#L26-L29

Reply:
Ah good to know.